### PR TITLE
Allow eslint check to be disabled

### LINKED ISSUE #6721: Disable eslint errors / improve dev settings when not using VSCode
**Setup**
- Rancher version:
- Browser type & version:

**Describe the bug**

**To Reproduce**
Disable Eslint plugin in VSCode
Add extra empty line spaces between code and save
Browser block the UI

**Result**

**Expected Result**

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by nwmac**
Fixes #6721

Adds support for a `SKIP_ESLINT` env var which will skip eslint check that blocks developing the UI when you have lint errors.

### CODE REVIEW

**Review by Shavindra — COMMENTED** [review]
Code works.
Although console still throws errors. I guess its expected.

File: shell/nuxt.config.js [hunk 1]
```diff
@@ -33,6 +33,9 @@ const pl = process.env.PL || STANDARD;
 const commit = process.env.COMMIT || 'head';
 const perfTest = (process.env.PERF_TEST === 'true'); // Enable performance testing when in dev
 
+// Allow skkipping of eslint check
+const skipEsLintCheck = (process.env.SKIP_ESLINT == 'true');
+
```
Comment (Shavindra) [inline shell/nuxt.config.js:38]: Lint failure...

**Review by Shavindra — CHANGES_REQUESTED** [review]

File: shell/nuxt.config.js (hunk 1 above)
Comment (nwmac) [reply shell/nuxt.config.js:38]: Fixed

**Review by Shavindra — DISMISSED** [review]
LGTM

**Pull request merged**